test_scripts/Polices/build_options/ATF_PTU_Trigger_IGN_cycles.lua

```diff
@@ -93,9 +95,10 @@ function Test:TestStep_SUCCEESS_Flow_PROPRIETARY()
 end
 
 --[[ Test ]]
+
 commonFunctions:newTestCasesGroup("Test")
 
-for i = 1, exchnage_after do
+for i = 1, exchange_after do
   Test["Preconditions_perform_" .. tostring(i) .. "_IGN_OFF_ON"] = function() end
 
   function Test:IGNITION_OFF()
```

Comment:
@dboltovskyi,
suggestion: Will be good if all tests in the loop have prefix/suffix, as example: 
Test["Preconditions_perform_" .. tostring(i) .. "_IGN_OFF_ON"]